### DISCUSSION TITLE
https://imagekit.io/blog/swiggy-imagekit-case-study/

### DISCUSSION THREAD

**Discussion by giscus** [imagekitio/discussions#8] [Blogs]
# https://imagekit.io/blog/swiggy-imagekit-case-study/

Learn how Swiggy improved media delivery for its apps with ImageKit, saving costs and enhancing user experience with advanced features.

https://imagekit.io/blog/swiggy-imagekit-case-study/

**Comment by Tanish-Varde** [comment]
It is quite inspiring as an engineer.
